site.yml

```diff
@@ -21,460 +21,460 @@ Documents:
         Slug: setup-caddy-proxy-with-minio
-        SEO Title:
-        SEO Description:
-        SEO Keywords:
+        SEO Title: MinIO | Setup Caddy proxy with MinIO - Cookbook/Recipe
+        SEO Description: In this recipe we will learn how to set up Caddy proxy with MinIO Server.
+        SEO Keywords: Caddy proxy
 
       - Setup Nginx proxy with Minio:
         Link: https://raw.githubusercontent.com/minio/cookbook/master/docs/setup-nginx-proxy-with-minio.md
         Slug: setup-nginx-proxy-with-minio
-        SEO Title:
-        SEO Description:
-        SEO Keywords:
+        SEO Title: MinIO | Setup Nginx proxy with MinIO - Cookbook/Recipe
+        SEO Description: In this recipe we will learn how to set up Nginx proxy with MinIO Server.
+        SEO Keywords: Nginx proxy
 
       - Store Apache Logs into Minio:
         Link: https://raw.githubusercontent.com/minio/cookbook/master/docs/aggregate-apache-logs-with-fluentd-and-minio.md
         Slug: aggregate-apache-logs-with-fluentd-and-minio
-        SEO Title:
-        SEO Description:
-        SEO Keywords:
+        SEO Title: MinIO | Store Apache Logs into MinIO - Cookbook/Recipe
+        SEO Description: In this recipe, we will learn how to use MinIO as log aggregator for fluentd using fluent-plugin-s3 plugin.
+        SEO Keywords: Apache
 
       - Rclone with Minio Server:
         Link: https://raw.githubusercontent.com/minio/cookbook/master/docs/rclone-with-minio.md
         Slug: rclone-with-minio-server
-        SEO Title:
-        SEO Description:
-        SEO Keywords:
+        SEO Title: MinIO | Rclone with MinIO Server- Cookbook/Recipe
+        SEO Description: This recipe describes how to use rclone with MinIO Server.
+        SEO Keywords: Rclone
 
       - Setup Apache HTTP proxy with Minio Server:
         Link: https://raw.githubusercontent.com/minio/cookbook/master/docs/setup-apache-http-proxy-with-minio.md
         Slug: setup-apache-http-proxy-with-minio-server
-        SEO Title:
-        SEO Description:
-        SEO Keywords:
+        SEO Title: MinIO | Setup Apache HTTP proxy with MinIO Server - Cookbook/Recipe
+        SEO Description: In this recipe we will learn how to set up Apache HTTP with mod_proxy module for connecting to MinIO Server. We are going to set up a new VirtualHost for example.com
+        SEO Keywords: Apache HTTP
 
       - Upload files from browser using pre-signed URLs:
         Link: https://raw.githubusercontent.com/minio/cookbook/master/docs/presigned-put-upload-via-browser.md
         Slug: upload-files-from-browser-using-pre-signed-urls
-        SEO Title:
-        SEO Description:
-        SEO Keywords:
+        SEO Title: MinIO | Upload files from browser using pre-signed URLs - Cookbook/Recipe
+        SEO Description: This guide describes how to use the presigned PutObject API from the MinIO JavaScript Library to generate a pre-signed URL.
+        SEO Keywords: pre-signed URLs
```

Comment:
This is a guide to upload files directly from browser using pre-signed URLs, so it might be a good idea to add some `upload file` related keywords as well